ta/arch/arm32/fix_ta_binary

```diff
@@ -27,7 +27,7 @@
 use strict;
 use warnings;
 use diagnostics;
-use Env qw($READELF);
+use Env qw($READELF $CROSS_COMPILE_ta);
```

Comment:
I would prefer if everything was `ta` since we don't have any other kind of TAs (the ones embedded in OP-TEE aren't really TAs). `ta_dev_kit.mk` should match what we have in the other makefiles so we'd better call it `user_ta` in `ta_dev_kit.mk` for now and have a discussion later whether we should do a `s/user_ta/ta/g`.